prime-router/docs/getting-started/fhir-functions.md

```diff
@@ -1,7 +1,68 @@
 # FHIR Functions
+FHIR functions are methods that can be run on a bundle in order to extract data from it (ex. retrieve an age or telephone area code)
+Currently, there are two places where FHIR functions are created. Both are in a file called CustomFhirPathFunctions and
```

Comment:
```suggestion
. Currently, there are two places where FHIR functions are created. Both are in a file called `CustomFhirPathFunctions` and
```

Reply:
Also they are NOT both in this file, correct? This file seems to only have `livdTableLookup`? I would probably reword everything after the first sentence, something like "There are two types of FHIR functions: internal and external. Internal functions are blah blah and external functions are blah blah" then you can talk about what files they relate to in their own sections and not up here in the general definition area.

Reply:
There are two files with that name and they are located in those.

Reply:
I can reword it to be more clear.